test/stages/stage1-vpc-subnet.tf

```diff
@@ -37,10 +37,9 @@ module "dev_priv_subnet" {
   #availability_zones = ["ap-south-1a","ap-south-1b"]
   subnet_cidrs       = var.priv_subnet_cidrs
   availability_zones = var.availability_zones
-  acl_rules          = var.acl_rules_pri
+  #acl_rules          = var.acl_rules_pri
```

Comment:
Same here... if not needed, can we get rid of these commented sections?

Reply:
Kept it as placeholder for testing with custom acls